src/main/java/org/myrobotlab/framework/repo/IvyWrapper.java

```diff
@@ -404,9 +405,9 @@ public void installDependency(String location, ServiceDependency library) {
   }
 
   @Override
-  synchronized public void install(String location, String[] serviceTypes) {
+  synchronized public void install(String location, String[] serviceTypes) throws IOException {
 
-    try {
+    // try {

```

Comment:
I think catching errors and providing a more helpful error message would be good for a critical component like `install()`. We can add a `System.exit()` in the catch to prevent attempts to continue with bad state.

Reply:
Ya, the nicest thing I can think about is mrl startsup (since it has all required packages to run Runtime, Security & WebGUI - it should never be thwarted by bad dependency install) ... When it starts up is should clearly show what dependencies have failed .. I think I made it show services not installed, but dunno all the details of its current behavior in that regard.